src/adapters/mc/ssh/LinuxSshServerSecurityBaseline_DeployIfNotExists.json

```diff
@@ -15,7 +15,7 @@
                 "version": "1.0.0",
                 "contentType": "Custom",
                 "contentUri": "https://github.com/Azure/azure-osconfig/releases/download/test_policy_package/LinuxSshServerSecurityBaseline.zip",

```

Comment:
Should we update the contentUri to point to ignite_2024 instead of test_policy_package?